topics/javascript/index.md

```diff
@@ -10,4 +10,4 @@ topic: javascript
 url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
 wikipedia_url: https://en.wikipedia.org/wiki/JavaScript
 ---
-JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a programming language running on the V8 JavaScript execution engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
+JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```

Comment:
Your edit makes sense, but now I am wondering why there is anything about `Node.js` in the description of the `JavaScript` topic at all?

I think I'd vote to remove that sentence altogether and add `nodejs` as a related topic (in my other suggestion).

```suggestion
JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. JS brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```